glean-core/ios/GleanTests/Scheduler/MetricsPingSchedulerTests.swift

```diff
@@ -110,8 +110,11 @@ class MetricsPingSchedulerTests: XCTestCase {
         let mps = MetricsPingScheduler()
         let now = Date()
 
+        Glean.shared.resetGlean(configuration: Configuration(), clearStores: true, uploadEnabled: false)
```

Comment:
Do you add `uploadEnabled: false` here so that no pings are uploaded?

Reply:
Yes, it's an added measure since the other tests in this test class handle initializing Glean in the way they need.  Adding comment now 👍